### ISSUE TITLE
Binary not available?

### ISSUE THREAD

**Issue by cedx** [cedx/which.php#5]
I install this package globally in order to be able to use the provided CLI executable:

```shell
$ composer global require cedx/which
```

Then, when I try to run the `which`, I get an error:

```shell
$ which
# Command not found
```

**Comment by cedx** [comment]
This is a packaging issue : the `composer.json` file lacks a `bin` field. It will be fixed in the next release.

**Comment by cedx** [comment]
New version released : v7.0.1.
